src/main/java/spoon/reflect/reference/CtIntersectionTypeReference.java

```diff
@@ -57,8 +56,4 @@
 
 	@Override
 	CtIntersectionTypeReference<T> clone();
-
-	@Override
-	@UnsettableProperty
-	<T extends CtActualTypeContainer> T setActualTypeArguments(List<? extends CtTypeReference<?>> actualTypeArguments);
```

Comment:
This property seems to be settable, because when I made it unset-able by 
```java
class CtIntersectionTypeReferenceImpl ... { ...
	public <C extends CtActualTypeContainer> C setActualTypeArguments(List<? extends CtTypeReference<?>> actualTypeArguments) {
		return (C) this;
	}
```
Then `TypeReferenceTest#testRecursiveTypeReference` started to fail.
But I am not able to decide yet what is correct. Test or Unsettable property. I do not understand spoon/java model around CtIntersectionTypeReference.

So check this one carefully.